packages/clients/textLocator/src/utils/coastalGazetteer/searchToponym.ts

```diff
@@ -3,6 +3,10 @@ import { Map } from 'ol'
 import { PolarStore, SelectResultFunction } from '@polar/lib-custom-types'
 import SearchResultSymbols from '@polar/plugin-address-search/src/utils/searchResultSymbols'
 import VectorSource from 'ol/source/Vector'
+import {
+  AddressSearchGetters,
+  AddressSearchState,
+} from '@polar/plugin-address-search/src/types'
```

Comment:
Please expose them correctly as mandated by this [ADR](https://github.com/Dataport/polar/wiki/Architecture-Decision-Records-%F0%9F%93%83#2024-03-05-how-to-expose-additional-exports-of-a-package)

Reply:
https://github.com/Dataport/polar/pull/163/commits/8a2ad1cf3fab8efd93e5201fd310bf59cc1c085c